index.js

```diff
@@ -332,7 +472,8 @@ instance.prototype.init_variables = function() {
 	variables.push({ name: 'num_dropped_frames', label: 'Number of dropped frames' });
 	variables.push({ name: 'num_total_frames', label: 'Number of total frames' });
 	variables.push({ name: 'preview_only', label: 'Preview only' });
-	variables.push({ name: 'recording', label: 'Recording state' });
+	// Line commented out due to creating a false state when recording started
+	//variables.push({ name: 'recording', label: 'Recording state' });
```

Comment:
What does this mean? Doesn't it work?

Reply:
I am not sure what is happening here and far from a great programmer, but if if line 

variables.push({ name: 'recording', label: 'Recording state' });

is included it breaks the recording function, and I have not yet found anything useful happening with this line of code included.  Anybody else have any ideas?

Reply:
@ewettach I ran your version and uncommented that line and it works fine on my system. Having that variable would be helpful. It starts out as "undefined" though, and doesn't get updated until a command is sent to start or stop recording, and then OBS websockets returns the status. I added a new function when it first connects to request the stream status, which returns booleans for both streaming and recording. If you will give me permissions to add to your fork, I can send that to you.

Reply:
@ewettach I added a HELP and README to your repo and also did a new PR on your repo to include my changes